app/buckets/handlers.js

```diff
@@ -42,10 +42,13 @@ exports.create_bucket = (req, res) => {
 exports.bucket_details = (req, res, next) => {
   Promise.all([Bucket.get(req.params.id), App.list(), User.list()])
     .then(([bucket, apps, users]) => {
+      const current_user_is_bucket_admin = bucket.has_admin(req.user.auth0_id);
```

Comment:
Since `bucket` is already passed to the template, you could just call `bucket.has_admin()` there instead of passing a new context variable.
I would be tempted to add a `is_bucket_admin(bucket)` method to `User` and just call it in the template like `current_user.is_bucket_admin(bucket)`, so that the logic in the template is `{% if current_user.is_bucket_admin() or current_user.is_superuser %}`